dist/angular-localForage.js

```diff
@@ -14,7 +14,11 @@
       return factory(angular, localforage);
     });
   } else if(typeof exports === 'object' || typeof global === 'object') {
-    module.exports = factory(angular, require('localforage')); // Node/Browserify
+    if(typeof module === 'undefined') {
+      global.module.exports = factory(angular, require('localforage')); // NW.js
+    } else {
+      modules.exports = factory(angular, require('localforage')); // Node/Browserify
```

Comment:
Thanks for fixing the typo!

I just mean do something like this on the command line (assuming macOS or linux):

```bash
$ cd LOCATION_OF_YOUR_FIXED_REPO
$ node
> var localForage = require('.')
```

Assuming it doesn't throw an error after that `require` call, we should be good. If you're still confused, I can do it tonight when I get home from work. To be honest, it might error out since node doesn't have a `LocalStorage` global, but I'm just trying to make sure it still works for Browserify and Electron which use the node-style `module.exports`.

Thanks again for the PR!